builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -175,6 +182,11 @@ func resourceAwsAutoscalingGroupCreate(d *schema.ResourceData, meta interface{})
 		autoScalingGroupOpts.HealthCheckGracePeriod = aws.Int64(int64(v.(int)))
 	}
 
+	if v, ok := d.GetOk("placement_group"); ok && v.(*schema.Set).Len() > 0 {
```

Comment:
USing availability_zones is not quite right for the placement_group. The `health_check_type` is a closer match to the system